### ISSUE TITLE
Import Wiki Pages to Web Site

### ISSUE THREAD

**Issue by anlumo** [Metalab/metalab.at#3]
- [ ] Automatically fetch the CoC and the Imprint pages from the wiki
- [ ] Reformat them to the site's color theme
- [ ] Add header and footer
- [ ] Include in the web site

**Comment by skauk** [comment]
Just out of curiosity. Did you ever consider embracing MediaWiki fully as a website engine?
There're plenty of themes out there to spice up its appearance.

**Comment by anlumo** [comment]
No, because it will always have hundreds of UI elements on the page that we don't want to have there. It's ok for an internal page, but not for external representation.

**Comment by skauk** [comment]
It's a pity though. MediaWiki is fully capable and customizable (e.g. Wikipedia's main page).
